action.yml

```
@@ -10,16 +10,38 @@ inputs:
     description: Devices to set up
     required: false
     default: ""
+  image-source-type:
+    description: How linux image is provided - url or path
+    required: false
+    default: "url"
+  image-source:
```

```suggestion
  kernel:
```
`image` is pretty generic, and `source` sounds like it wants source code.

---

It is not a kernel, but whole linux image with busybox, bootloader, tools, etc.